client/store/current-user/actions.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import apiRequest from '@wordpress/api-request';
```

Comment:
[`api-request` has been deprecated](https://github.com/WordPress/gutenberg/commit/247a6e07c22792769c4d397ad3a6d118dbc90a77) in favor of [api-fetch](https://github.com/WordPress/gutenberg/tree/master/packages/api-fetch) (though you can also POST with fetch)